src/models/thelio-major-r5-n3/repairs.md

```diff
@@ -0,0 +1,641 @@
+
+5. Slide the CPU carrier tray out of the CPU bracket.
+    - Be careful not to bend any of the gold pins on the CPU socket, and do not touch the gold pads on the CPU.
+
+![CPU (removed)](./img/cpu-removed.webp)
+
+6. Slide the new CPU's carrier tray into the CPU bracket.
+    - AMD Threadripper CPUs come with carrier trays, so removing the CPU from the tray is generally not necessary.
+7. Flip the CPU bracket back down onto the socket, gently pressing the blue tabs to seat the CPU carrier tray onto the motherboard.
+8. Flip the CPU bracket cover back down.
+9. Screw the three Torx screws back into place.
+    - The AMD Threadripper-branded tool applies 1.5 Nm of torque. When the screw is tight enough, it will give way.
+    - If you're using a different screwdriver, use a torque tool to apply approximately the same amount of torque (1.5 Nm, or ~13.275 in-lb).
+    - If you're using a screwdriver without a torque tool, try to tighten all three screws evenly. After re-assembling the computer, [run a RAM test](https://support.system76.com/articles/hardware-failure#memory) to ensure the CPU is secured with the correct amount of force.
+
+### Steps to install the thermal paste/CPU cooler:
+
+1. Draw an `X` shape of thermal paste onto the CPU.
+
+![Thermal paste application](./img/thermal-paste-application.webp)
+
+2. Place the CPU cooler onto the CPU; while holding it in place, screw its vertical mounting bar into the horizontal mounting brackets.
+    - The cooler should be oriented so the Thermalright logos are on the bottom.
+    - Insert each screw partially, then finish tightening both.
+3. Set the computer upright, then reinstall the cooler-mounted CPU fan, CPU duct, GPU, side brace, and top case.
+
+## Replacing the RAM:
+
+Thelio Major R5-N3 supports up to 512GB (4x128GB) of RAM. The RAM sticks are DDR5 ECC DIMMs running at a speed of 5600 MHz.
+
+If you've purchased new RAM, need to replace your RAM, or are reseating your RAM, follow these steps.
+
+**Tools required:** Cross-head (Phillips) screwdriver  
+**Time estimate:** 15 minutes  
+**Difficulty:** Medium <span style="color:orange;">●</span>
+
+### Steps to replace the RAM:
+
+1. Follow the steps above to [remove the top case](#removing-the-top-case), [remove the CPU duct](#removing-the-cpu-duct), and [remove the CPU cooler](#steps-to-remove-the-cpu-coolerthermal-paste).
+    - Removing the CPU cooler is optional, but is recommended to provide easier access to the RAM slots.
+    - Replacing the thermal paste is recommended if the CPU cooler is removed.
+2. To remove an existing RAM stick, flip the left and right latches away from the stick, then pull the stick out of the slot.
+
+![RAM slots](./img/ram-slots.webp)
+
+3. Make sure the tabs on the top and bottom of the slot are open (pulled away from the slot), then insert the new RAM (or re-seat the existing RAM) into the slot.
+    - The RAM stick will only fit in one direction. The larger group of pins goes on top.
```

Comment:
Should "top" be changed to either left or right, since the RAM slots are horizontal?